### PR TITLE
added uigetcheckboxstate command

### PR DESCRIPTION
**Pull request by danieladamkaiser**

### CODE REVIEW

File: G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using G1ANT.Addon.UI.Api;
+using G1ANT.Addon.UI.Structures;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.UI.Commands
+{
+    [Command(Name = "ui.getcheckboxstate",
+        Tooltip = "This command returns targeted checkbox's state")]
```
Comment (przegwar) [inline G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs]: I think this attribute should be one-liner

File: G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    public class UIGetCheckboxStateCommand : Command
+    {
+        public class Arguments : CommandArguments
+        {
+            [Argument(Required = true, Tooltip = "Desktop application checkbox")]
```
Comment (przegwar) [inline G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs]: I would suggest something like "WPath to checkbox" or something more related to what this argument is all about

File: G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        }
+
+        public UIGetCheckboxStateCommand(AbstractScripter scripter) : base(scripter)
+        {
+        }
+
+        public void Execute(Arguments arguments)
+        {
+            var element = UIElement.FromWPath(arguments.WPath);
+            if (element != null)
```
Comment (wojszub) [inline G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs]: unnecessary check, FromWPath() will throw exception if can't find element I think

File: G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
+﻿using G1ANT.Addon.UI.Api;
+using G1ANT.Addon.UI.Structures;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.UI.Commands
+{
+    [Command(Name = "ui.getcheckboxstate", Tooltip = "This command returns targeted checkbox's state")]
+    public class UIGetCheckboxStateCommand : Command
```
Comment (wojszub) [inline G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs]: for me this command is not consistent, command name and tooltip suggest that command is for Checkbox control only, I would like to split code to separate command to check Radio Button control but I know that you make this exactly what was written in Task, consult this with Przemek Witusik (he was a reviewer previously :))

File: G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs (hunk 3 above)
Comment (danieladamkaiser) [reply G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs]: Well, yeah. You are right

File: G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs (hunk 4 above)
Comment (danieladamkaiser) [reply G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs]: I agree. I will verify with @pwitusik  before merging

Comment (pwitusik) [reply G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs]: OK, I agree name of the command is confusing. I don't want to split the same functionality to different commands, I prefer to change the name of the command to be common for checkboxes and radio buttons. Any suggestions for the different name?

**Pull request merged**